nitransforms/io/itk.py

```diff
@@ -89,33 +155,74 @@ def __init__(self,
                  check=True):
         """Initialize with (optionally) a list of transforms."""
         super().__init__(binaryblock, endianness, check)
-        self._xforms = []
-        for i, mat in enumerate(xforms or []):
-            xfm = ITKLinearTransform()
-            xfm['parameters'] = mat
-            xfm['index'] = i + 1
-            self._xforms.append(xfm)
+        self.xforms = [ITKLinearTransform(parameters=mat)
+                       for mat in xforms or []]
+
+    @property
+    def xforms(self):
+        """Get the list of internal ITKLinearTransforms."""
+        return self._xforms
+
+    @xforms.setter
+    def xforms(self, value):
+        self._xforms = value
```

Comment:
we should check or coerce into list here?
```suggestion
        self._xforms = list(value)
```